packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -17,7 +17,9 @@ export const warnAboutSpreadingKeyToJSX = __VARIANT__;
 export const disableInputAttributeSyncing = __VARIANT__;
 export const enableFilterEmptyStringAttributesDOM = __VARIANT__;
 export const enableLegacyFBSupport = __VARIANT__;
-export const enableDebugTracing = !__VARIANT__;
+// TODO: These features do not currently exist in the new reconciler fork.
+export const enableDebugTracing = !__VARIANT__ && __PROFILE__;
+export const enableSchedulingProfiler = !__VARIANT__;
```

Comment:
Whoops, looks like I misspoke on my previous comment. My apologies. I meant to suggest:
```suggestion
// TODO: These features do not currently exist in the new reconciler fork.
export const enableDebugTracing = !__VARIANT__;
export const enableSchedulingProfiler = !__VARIANT__ && __PROFILE__;
```